Header header
int32 signal
uint8 signal_raw
uint8 fall_count
int8 finger_clutch
uint8[3] acc
float64 acc_abs
string serial
